wordpress-mobile/WordPress-Android
# Feature/global notification setting

### Fix
Add a global notification preference to the ***Notification Settings*** screen under the ***Sights and Sounds*** section to allow users to enable/disable notifications on their device.  See the screenshots below for illustration.

![notification_settings](https://user-images.githubusercontent.com/3827611/30082538-688648c8-9248-11e7-9725-59582002cfc4.png)

### Test
1. Go to ***Me*** tab.
2. Tap ***Notification Settings*** option.
3. Notice ***Receive notifications*** setting.
4. Notice ***Receive notifications*** setting is on.
5. Notice all ***Sights and Sounds*** settings are enabled.
6. Like a post from another account.
7. Notice notification is received.
8. Tap ***Receive notifications*** setting.
9. Notice ***Receive notifications*** setting is off.
10. Notice all ***Sights and Sounds*** settings are disabled.
11. Like a post from another account.
12. Notice notification is not received.
